external-crates/move/crates/move-compiler/src/hlir/ast.rs

```diff
@@ -144,11 +144,11 @@ pub type FunctionBody = Spanned<FunctionBody_>;
 
 #[derive(PartialEq, Debug, Clone)]
 pub struct Function {
+    pub loc: Loc,
```

Comment:
Is there a motivation for this sort of churn?

Reply:
Rebase. Some of these weren't here at the time of writing this originally... this PR is real old. 
I can try to revert them if you want